src/js/common/schemaform/definitions/nonMilitaryJobs.js

```diff
@@ -0,0 +1,24 @@
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
+
+import EmploymentPeriodView from '../../../edu-benefits/components/EmploymentPeriodView';
+
+export const schema = commonDefinitions.nonMilitaryJobs;
```

Comment:
Actually removing this import causes the tests to fail. Should I import the schema in the tests alone so that passes, are just keep it here?